src/cow-react/api/gnosisProtocol/api.ts

```diff
@@ -381,8 +381,16 @@ function _mapNewToLegacyParams(params: FeeQuoteParams): QuoteQuery {
     validTo,
     partiallyFillable: false,
     priceQuality,
+    // TODO: check this setup as this is for all other orders
```

Comment:
onchainOrder and signingScheme are optional. You can just leave them "unset" for the normal usecase.

Reply:
👍